### PR TITLE
Chapter 11: Blade Template

### PR DESCRIPTION
**Pull request by Hungnd-0370**

### CODE REVIEW

**Comment by Hungnd-0370** [comment]
ready

File: .gitignore [hunk 1]
```diff
@@ -14,4 +14,5 @@ yarn-error.log
 /.idea
 /.vscode
 bower_components
-public/css
+public/css/*
+public/js/*
```
Comment (ghost) [inline .gitignore]: eof

File: resources/scss/app.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    box-shadow: inset 0 0 0 9999px #d9d9d9;
+}
+
+.table>tbody {
+    vertical-align: inherit;
+}
+
+.table>thead {
+    vertical-align: bottom;
+}
```
Comment (ghost) [inline resources/scss/app.scss]: eof

File: webpack.mix.js [hunk 3]
```diff
@@ -15,3 +15,4 @@ mix.js('resources/js/app.js', 'public/js').postCss('resources/css/app.css', 'pub
     require('tailwindcss'),
     require('autoprefixer'),
 ]);
+mix.sass('resources/scss/app.scss', 'public/css');
```
Comment (ghost) [inline webpack.mix.js]: eof

**Comment by Hungnd-0370** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**